.travis/install.sh

```diff
@@ -29,5 +30,35 @@ fi
 
 pip install -e ./pulpcore-plugin
 
+if [ -n "$PULP_SMASH_PR_NUMBER" ]; then
+  pip uninstall -y pulp-smash
+  git clone https://github.com/PulpQE/pulp-smash.git
+  pushd pulp-smash
+  git fetch origin +refs/pull/$PULP_SMASH_PR_NUMBER/merge
+  git checkout FETCH_HEAD
+  popd
+  pip install -e ./pulp-smash
+fi
+
+# Install Podman client
+sudo add-apt-repository -y ppa:projectatomic/ppa
+sudo apt-get update -y
+sudo apt-get install -y podman
+
+# Configure podman for insecure registry
+sudo bash -c 'cat << EOF > /etc/containers/registries.conf
+[registries.search]
+registries = ["docker.io", "registry.fedoraproject.org", "quay.io", "registry.access.redhat.com", "registry.centos.org"]
+
+[registries.insecure]
+registries = ["localhost:8080", "localhost:8000"]
```

Comment:
what's is the reason of having localhost on  8000?

Reply:
well no need.. it was just a copy/paste.. fixing it